app/src/main/java/com/example/app/logout.java

```diff
@@ -0,0 +1,20 @@
+package com.example.app;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+
+public class logout extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_logout);
+    }
+
+    public void openLogin(View view) {
+        startActivity(new Intent(this,login.class));
+    }
+}
```

Comment:
Is this the function for the logout?

Reply:
We used kasi an online database to authenticate our users, this will work on our end since this only applies sa offline database sa android (sqlite)